docs/readmes/lte/ha_setup.md

```diff
@@ -0,0 +1,179 @@
+`Add Storage`.
+6. On page `Add Storage`, use default of 8gb. Proceed to `Add Tags`.
+7. On page `Add Tags`, optionally add tags to identify this as a secondary
+Magma AGW. Proceed to `Configure Security Group`.
+8. On page “Configure Security Group”, create a new security group with the
+rules listed below. It is advised to limit the source IPs to the subnet that i
+the primary gateway resides in for all rules other than SSH. Proceed to
+`Review and Launch`.
+
+|Type	|Protocol	|Port Range	|Source	|Description	|
+|---	|---	|---	|---	|---	|
+|SSH	|TCP	|22	|0.0.0.0/0	|-	|
+|SCTP (132)	|SCTP (132)	|All	|0.0.0.0/0	|-	|
+|Custom TCP	|TCP	|3386	|0.0.0.0/0	|-	|
+|All UDP	|UDP	|0 - 65535	|0.0.0.0/0	|	|
+|All ICMP - IPv4	|ICMP	|All	|0.0.0.0/0	|-	|
+
+1. Review that the selected settings are as described here. Then proceed to
+`Launch`.
+2. Select `Create a new key pair`, then save the key pair created to your host
+machine. This pair will be used to access the gateway, so ensure the pair is
+saved in a safe and durable location.
+3. Finish by selecting `Launch Instances`.
+
+### ENI Configuration
+
+Before installing Magma, we will add a second interface to gateway by creating
+an ENI and attaching it to the EC2 instance.
+
+1. In the EC2 service on AWS, navigate to the `Network Interfaces` section
+under the `Network and Security` tab on the side panel.
+2. Select `Create network interface` in the upper right corner.
+3. On the `Create network interface` configuration page, select the subnet for
+the ENI. To work properly, this subnet must cannot be the same subnet the that
+EC2 instance was deployed with. These subnets must be in the same availability
+zone though.
+4. Select the same subnet that was used to deploy the EC2 instance.
+5. Once configured, select `Create network interface`.
+6. Navigate to the EC2 instances page.
+7. Find the recently deployed EC2 instance on the left hand side. Then select
+`Actions` → `Networking` → `Attach network interface`.
+8. On page `Attach network interface`, select the recently created ENI and then
+click `Attach`.
+
+### Install Magma
+
+1. Find the public IP for the gateway instance by navigating to `Instances` on
+the AWS EC2 service. Select the instance and copy the `Public IPv4 Address` in
+the instance summary.
+2. Add the AWS gateway key that was created when the instance was launched:
+`ssh-add ~/.ssh/aws_key.pem`
+3. SSH to EC2 instance using the public IP from step 1:
+`ssh admin@<instance_public_ip>`
+4. Now install Magma
+
+```
+[admin@<public_ip>~/] sudo su
+[root@<public_ip>:/home/admin] wget https://raw.githubusercontent.com/facebookincubator/magma/v1.4/lte/gateway/deploy/agw_install.sh
+[root@<public_ip>:/home/admin] bash agw_install cloud
+`
+```
+
+When  you see "AGW installation is done." It means that your AGW installation
+is done, you can make sure magma is running by executing:
+
+```
+service magma@* status
+```
+
+### Access Gateway Configuration
+
```

Comment:
Since this is going to be running the HA AGWs in AWS, why not an AMI that has the AGW built-in? This will make the deployment of the AGW process instantaneous. Code available in fbcinternal/mantle or https://github.com/magma/magma/pull/4718 which works for 1.3